src/styles/footer.scss

```diff
@@ -0,0 +1,150 @@
+.footer {
+  background-color: $footer-background;
+  color: $footer-text-collor;
+  padding: 30px 20px 40px;
+
+  @include tablet {
+    padding: 30px 27px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    column-gap: 30px;
+  }
+
+  @include desktop {
+    column-gap: 120px;
+    padding: 60px 50px;
+    max-width: 1280px;
+  }
+
+  @include big-desktop {
+    padding: 60px 130px;
+    max-width: 1440px;
+  }
+
+  &__menu {
+    display: flex;
+    flex-direction: column;
+    gap: 11px;
+    font-weight: 500;
+    font-size: 18px;
+    margin-bottom: 20px;
+
+    @include tablet {
+      margin-bottom: 50px;
+    }
+
+    @include desktop {
+      gap: 40px;
+      position: relative;
+      flex-direction: row;
+      margin-bottom: 20px;
+    }
+  }
+
+  &__item {
+    height: 100%;
+
+    @include desktop {
+      &:hover .footer__link::after {
+        opacity: 1;
+        transform: scaleX(1);
+      }
+    }
+  }
+
+  &__contacts {
+    font-size: 14px;
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin-bottom: 40px;
+    max-width: 280px;
+
+    @include tablet {
+      max-width: 100%;
+    }
+
+    @include desktop {
+      gap: 20px;
+      margin-bottom: 30px;
+    }
+  }
+
+  &__socials {
+    display: flex;
+    gap: 20px;
+    margin-bottom: 40px;
+
+    @include desktop {
+      gap: 14px;
+      position: absolute;
```

Comment:
Avoid using absolute positioning for the social links, it may cause issues with different screen sizes. Consider using flexbox or grid for positioning.